Adyen/UI/Form/FormViewItemManager.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2020 Adyen N.V.
+// Copyright (c) 2021 Adyen N.V.
```

Comment:
The are so many files with only header changed 🤔

Reply:
yeah, I were experimenting a lot :-D, so I'm changed lots of files, the reverted.